Assets/MRTK/SDK/Profiles/HoloLens2/DefaultHoloLens2InputSystemProfile.asset

```diff
@@ -146,7 +146,7 @@ MonoBehaviour:
   enableControllerMapping: 1
   controllerMappingProfile: {fileID: 11400000, guid: 39ded1fd0711a0c448413d0e1ec4f7f3,
     type: 2}
-  controllerVisualizationProfile: {fileID: 11400000, guid: 345c06fdf3732db46b96299bd3cba653,
+  controllerVisualizationProfile: {fileID: 11400000, guid: ac2477ca3c42e514f86bc810089ba605,
```

Comment:
Does the HoloLens 1 profile need to be updated as well?

Reply:
Yeah, can do!